Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -31,4 +32,18 @@ extension FileManager: LintableFileManager {
             return nil
         } ?? []
     }
+
+    public func isFile(atPath path: String, modifiedSince: Date) -> Bool {
+        // If there is any problem reading the modification date, prefer to lint the file
+        do {
+            let attr = try attributesOfItem(atPath: path)
+            if let fileModificationDate = attr[FileAttributeKey.modificationDate] as? Date {
+                return fileModificationDate > modifiedSince
```

Comment:
Actually I commented on this earlier. I think the date comparison should be != and not >. It's always possible to reset the clock to an earlier time, on purpose or by accident. I think some of your other logic checking for inconsistent last run date could probably be removed if you simply linted the file if the dates don't match.